.gitlab-ci.yml

```diff
@@ -0,0 +1,69 @@
+.default: &default
+  only:
+    - merge_requests
```

Comment:
remove this file

Reply:
added this for testing purpose, will remove once end-to-end testing is done

Reply:
removed this file, created separated branch for testing